src/Test/Perf/helloworld/HelloWorld.csx

```diff
@@ -0,0 +1,20 @@
+#load "../util.csx"
+using System.IO;
+
+var pathToCsc = Path.Combine(BinDebugDirectory(), "csc.exe");
+var pathToHelloWorld = Path.Combine(MyWorkingDirectory(), "HelloWorld.cs");
+var pathToOutput = Path.Combine(MyArtifactsDirectory(), "HelloWorld.exe");
+
+ProcessResult result;
+
+var msToCompile = WalltimeMs(out result,
```

Comment:
Having iterate do timing seems reasonable.  But yeah, we'll have to do some really weird stuff when it comes to doing typing-speed tests in vs.